src/subscription_manager/gui/data/glade/register_dialog.glade

```diff
@@ -20,12 +20,13 @@
             <property name="layout_style">end</property>
             <child>
               <object class="GtkButton" id="cancel_button">
-                <property name="label" translatable="yes">Close</property>
+                <property name="label">gtk-close</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="receives_default">True</property>
+                <property name="use_stock">True</property>
                 <child internal-child="accessible">
-                  <object class="AtkObject" id="close_button-atkobject">
+                  <object class="AtkObject" id="cancel_button-atkobject">
```

Comment:
Hmm. Can't remember if that was glade weirdness or something I did on purpose.

Taking a look to see what I intended...